### ISSUE TITLE
Active links and read link colors not colorblind friendly

### ISSUE THREAD

**Issue by srirambv** [brave/browser-laptop#7842]
From community  https://community.brave.com/t/active-links-and-read-link-colors-not-colorblind-friendly/1787
>cannot see any difference in the colors used for an active click-able link I have never visited, and one I have. It would be very helpful if the default colors were shifted farther apart (visited links darker than unvisited links by at least 10%). 

cc: @bradleyrichter 

Setting temp milestone for now.

**Comment by bradleyrichter** [comment]
We have not yet implemented link coloring, partially due to a security risk. 

original issue: https://github.com/brave/browser-laptop/issues/512

**Comment by diracdeltas** [comment]
actually there was no security risk in implementing link coloring. it's implemented now but the colors are kind of hard to see. they appear to be the same as in chrome though.

**Comment by bradleyrichter** [comment]
I stand corrected and happy.

Let's beat Chrome with a color-blind friendly delta.

If we move towards a turquoise blue, the effect is much more visible in all 3 color blindness types:

![image](https://user-images.githubusercontent.com/13509546/27111333-4aa90370-5064-11e7-8fa0-b74587d5b9f3.png)

I suspect the blue-to-purple standard was chosen to prevent adding extra attention to links unfairly even though you had already visited them. If this becomes a problem, we may need to add color-blindness accessibility mode.


visible delta with green-blue adjustment shown in 3 major colorblindness tests:

![image](https://user-images.githubusercontent.com/13509546/27111391-abd0f2e8-5064-11e7-8a45-dd1a98db11dd.png)

![image](https://user-images.githubusercontent.com/13509546/27111412-c068e4c2-5064-11e7-905c-dcee8d0f7524.png)


![image](https://user-images.githubusercontent.com/13509546/27111418-cb989b76-5064-11e7-8873-96ec1f7cec9c.png)

**Comment by bradleyrichter** [comment]
the magic color for visited links shown in the images above is: #00997c

**Comment by jamesray1** [comment]
Related: everything that was blue in the previous latest stable version that I was using is now purple with 0.19.16.

**Comment by bsclifton** [comment]
Closing in favor of https://github.com/brave/brave-browser/issues/907
